README.md

```diff
@@ -1 +1,12 @@
-This software includes third party open source software component: `Ethereum Foundation`. It has its own license. Please see `.3rd_parth_licenses/Ethereum/LICENSE`. The source is from [tests](https://github.com/ethereum/tests) including `GeneralStateTests` and `VMTests` folders.
+# The Test Data Used by CITA
```

Comment:
By or by?

Reply:
https://english.stackexchange.com/questions/14/which-words-in-a-title-should-be-capitalized